_posts/henry_zhao/2019-03-06-Implement-Stardust.md

```diff
@@ -0,0 +1,287 @@
+---
+layout: post
+title: 'Accelerate data visualization with WebGL Platform Stardust'
+breadcrumb: true
+author: henry_zhao
+date: 2019-03-06
+categories: henry_zhao
+tags:
+  - Javascript
+  - WebGL
+  - Stardust
+teaser:
+  info: Data visualization with massive data could be fine in your browser
+  image: henry_zhao/stardust.png
+header:
+  version: small
+  title: Software Engineering Blog
+  image: header-logo-crop.png
+  icon: icon-blog
+---
+
+<image src="{{ site.urlimg }}/henry_zhao/future_world.jpg" />
+
+# Introduction
+
+Genomic data is huge and messy. In order to provide researchers a powerful data analysis&management platform, what we present data is the most popular data processing library, d3. It has a wealth of tools that can use svg to portray and manipulate data patterns to the greatest extent possible with different ranges and different forms of pie chart, histograms charts and human figures. For example:
+
+<image src="{{ site.urlimg }}/henry_zhao/human_body.png" />
+
+However, when the data set is too big(larger than 10000 node), there will an inevitable DOM operation bottleneck. It makes me think about what kind of tools can improve or even replace d3 to be able to visualize data faster.
+So first we should know what is svg and why it has bottleneck.
```

Comment:
`So first we should understand what SVG is, and what are the possible causes of a bottleneck.`